packages/dx-react-demos/src/demo-sources/grid-grouping/disable-column-grouping.jsxt

```diff
@@ -0,0 +1,63 @@
+import React from 'react';<%&additionalImports%>
+import {
+  GroupingState,
+  IntegratedGrouping,
+} from '@devexpress/dx-react-grid';
+import {
+  Grid,
+  Table,
+  TableHeaderRow,
+  TableGroupRow,
+  GroupingPanel,
+  DragDropProvider,
+  Toolbar,
+} from '@devexpress/dx-react-grid-<%&themeName%>';
+
+import {
+  generateRows,
+} from '../../../demo-data/generator';
+
+const groupingStateColumnExtensions = [
```

Comment:
Why do we extract a constant for extensions while columns is inlined into a state?